src/mixins/filing-template-mixin.ts

```diff
@@ -519,10 +522,9 @@ export default class FilingTemplateMixin extends DateMixin {
     this.setFoundingDate(draftFiling.business.foundingDate)
 
     // restore Restoration data
-
-    // *** TODO: restore date
-    // *** TODO: restore type
-    // *** TODO: restore expiry
+    this.setRestorationDate(draftFiling.restoration.date)
+    this.setRestorationType(draftFiling.restoration.type)
+    this.setRestorationExpiry(draftFiling.restoration.expiry || null)
```

Comment:
I had to decide whether to do this, or this:
```
this.setRestoration({
   date: draftFiling.restoration.date,
   type: draftFiling.restoration.type,
   expiry: draftFiling.restoration.expiry || null
})
```

I chose the former because we're going to need those setters anyway, since those 3 properties are set from different components.